src/app/component/shared/menu/menu.component.html

```diff
@@ -24,6 +24,7 @@ <h6 class="nav-link disabled m-0">
             [id]="menuType + '-tooltip-' + i"
             [link]="link"
             [placement]="placement"
+            [tooltip]="link.tooltip(user)"
```

Comment:
Would it have been far simpler to just pass the user object to component instead of pre-calculating the tool-tips? Bit late to change back now, however this should be changed back someday to reduce redundancy and simplify unit tests.